internal/mongodoc/doc.go

```diff
@@ -121,8 +121,21 @@ type Entity struct {
 	Development bool
 }
 
-func (e *Entity) CanonicalURL() *charm.URL {
-	return e.URL
+// CanonicalURLs returns the canonical URLs that can be used
```

Comment:
CanonicalURLs seems like the wrong name now. Surely there is only one canonical URL which is _id. Also this description is quite unhelpful. cs:mysql can be used to look up an entity, but it's not (presumably) canonical. Where as cs:trusty/mysql-10 is presumably meant to be canonical, but I can't use FindID to retrieve one. What does this function now do, and why?